tasks/frontEnd/src/App.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <AppProvider>
+    <Page title="Judge.Me Reviews">
+      <Layout>
+        <LayoutSection>
+          <Button @click="toggleCollapsibleActive" style="margin-bottom: 20px;">{{ formButtonMessage }}</Button>
```

Comment:
Any reason for opting to use inline styles here?

Reply:
No reason other than it was the only instance, would usually avoid inline styles unless necessary for production code